gcloud-java-contrib/gcloud-java-nio-examples/src/main/java/com/google/gcloud/nio/examples/ListFilesystems.java

```diff
@@ -28,5 +43,4 @@ private static void listFilesystems() {
       System.out.println("  " + p.getScheme());
     }
   }
-
-}
+}
```

Comment:
Ah, that's what I get for removing all empty lines in sight when I wasn't sure which empty line you didn't like. OK, put that one back in.